src/components/Textarea.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Textarea = ({
+  placeholder, register, name, ...htmlAttributes
+}) => (
+  <div {...htmlAttributes}>
+    <div className="relative w-full min-w-[200px]">
+      <textarea
+        {...register(name)}
+        className="peer h-full min-h-[100px] w-full resize-none rounded-[7px] border border-gray-400 border-t-transparent bg-transparent px-3 py-2.5 font-sans text-sm font-normal text-blue-gray-700 outline outline-0 transition-all placeholder-shown:border placeholder-shown:border-blue-gray-200 placeholder-shown:border-t-gray-400 focus:border-2 focus:border-green-400 focus:border-t-transparent focus:outline-0 disabled:resize-none disabled:border-0 disabled:bg-blue-gray-50"
```

Comment:
[OPTIONAL] these class names are pretty long, consider using styled components or variables for long className, it makes code much cleaner.